website/src/docs/thumbnail-generator.md

```diff
@@ -77,6 +78,10 @@ If both width and height are given, only width is taken into account.
 >
 > See issue [#979](https://github.com/transloadit/uppy/issues/979) and [#1096](https://github.com/transloadit/uppy/pull/1096) for details on this feature.
 
+### `thumbnailtype: 'image/jpeg'`
+
+MIME type of the resulting thumbnail. Default thumbnail MIME type is `image/jpeg`. 
```

Comment:
Maybe we should elaborate on why it might be useful to change the MIME type? PNG, transparency.

Reply:
Added a note about transparency.